mlrun/track/__init__.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# flake8: noqa  - this is until we take care of the F401 violations with respect to __all__ & sphinx
+
+from ..utils import logger
+from .base_tracker import BaseTracker
+from .tracker import Tracker
+from .tracker_manager import TrackerManager
+from .trackers import MLFlowTracker
+
+# Add a tracker to this list for it to be added into the global tracker manager:
+_AVAILABLE_TRACKERS = [MLFlowTracker]
+trackers_manager = TrackerManager()
```

Comment:
A global variable should be written in all caps, and to mark the user it should not be accessed, add a `_` prefix to it (it should be accessed via the `get_trackers_manager` function)

```suggestion
_TRACKERS_MANAGER = TrackerManager()
```